content/docs/tech-insights/track-sonarqube/index.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Track On-prem Sonarqube metrics
+publishedDate: '2023-08-04T15:15:00.0Z'
+description: How to track and visualize a Sonarqube metrics with Roadie
+---
+
+One of the most advantageous steps a platform team can take to support application developers (app devs) is to offer them a reliable and secure platform for enhancing code quality. This is often achieved by incorporating SonarCloud into the development process, enabling teams to identify and rectify code-related issues effectively.
+
+By tracking issues and hotspots reported by Sonarqube across your organization allows you to highlight and to proactively address potential pitfalls, enhance code integrity, and bolster application security. This practice not only fosters a culture of continuous improvement but also ensures that software projects evolve with a heightened focus on robustness and reliability.
+
+With Roadie’s Tech Insights feature, you can monitor issues and hotspots right within your software catalog:
```

Comment:
This is true, but really what we want the user to learn here is that they can connect it to on-prem software. So we should mention that.

Reply:
Let me know what you think of the new text.